base.yaml

```diff
@@ -15,6 +16,13 @@ ratgdo:
   output_gdo_pin: ${uart_tx_pin}
   input_obst_pin: ${input_obst_pin}
   remote_id: 0x539
+  on_sync_failed:
+    then:
+      - homeassistant.service:
+          service: persistent_notification.create
+          data:
+            title: "RATGDO sync failed"
+            message: "Failed to communicate with garage opener on startup, check rolling code counter and restore from history. [esphome devices](/config/devices/dashboard?domain=esphome)""
```

Comment:
Added link to esphome devices here for convenience.  Is there a way to have a link do ratgdo directly?

Reply:
I think the best we can do is the `domain
```javascript
        this._searchParms.get("config_entry") ||
        this._searchParms.get("domain")
```